README.md

```diff
@@ -141,4 +142,4 @@ See docs/Testing.rst.
 
 ## Contributing to Swift
 
-Contributions to Swift are welcomed and encouraged! Please see the [Contributing to Swift guide](https://swift.org/contributing/).
```

Comment:
Also, was there any change here?  Was it just a whitespace change?  I can't tell in the diff.